src/JBrowse/Store/SeqFeature/BAM/LazyFeature.js

```diff
@@ -100,7 +101,7 @@ var Feature = Util.fastDeclare(
     },
 
     id: function() {
-        return this._get('name')+'/'+this._get('md')+'/'+this._get('cigar')+'/'+this._get('start')+'/'+this._get('multi_segment_next_segment_reversed');
+        return Crc32.crc32_raw(this.bytes.byteArray.slice(this.bytes.start, this.bytes.end));
```

Comment:
do you think it would be worth it performance-wise to avoid the `slice` and instead make crc32_raw accept bounds for the calculation?  e.g. `crc32_raw(this.bytes.byteArray, this.bytes.start, this.bytes.end)`

Reply:
also, is `id()` being memoized? We need to make sure this only gets calculated once per feature.

Reply:
id() is memoized, the _get() arguments stores all lazyily parsed attributes in this.data and does not again parse it

also I changed the crc to avoid slice now :)